.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,16 +10,36 @@ assignees: ''
 I tried to [...] and I expected [...] instead [...]
 
 **GEOPM version**
-E.g. output from `geopmread --version`
+Output from `geopmread --version`
 
 **Expected behavior**
 A clear and concise description of what you expected to happen.
 
 **Actual behavior**
-A clear and concise description of what actually happened
+A clear and concise description of what actually happened.
 
 **Error messages produced**
-If possible use `configure --enable-debug ...` for more verbose error messaging.
+Any output to standard error or standard output that may be related to
+the bug.  Was debug messaging was enabled at build time?
+
+_Note_:
+
+If possible use the `--enable-debug` configuration option when
+building GEOPM.  This will produce more verbose error messaging at run
+time. To enable this when using the build script in the base directory
+run the following command:
+
+```bash
+GEOPM_GLOBAL_CONFIG_OPTIONS="$GEOPM_GLOBAL_CONFIG_OPTIONS --enable-debug" \
```

Comment:
```suggestion
GEOPM_GLOBAL_CONFIG_OPTIONS="--enable-debug" \
```

Nothing else is setting this env option presently.  Not sure what else would be.  This is the syntax that the help text of the build script recommends.